### DISCUSSION TITLE
Adding actual dynamic playlist to app would be cool

### DISCUSSION THREAD

**Discussion by Rupamhere1** [FreeTubeApp/FreeTube#6432] [General]
I'm new to this app, so i'm not sure if it can be done or not but wouldn't it be great if we are able to save a dynamic playlist.

I mean what we currently have now is that we create a playlist and add videos to it or import 100 videos from an existing playlist on YouTube. but some playlist like [this](https://youtube.com/playlist?list=PLpuxPG4TUOR4k3h0mRP8j-bU5BTSejsbI) are updated daily as new videos are added by the owner. in such scenario saving the playlist in app becomes useless as it doesn't automatically add new video added by the owner. can something be done about it?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @silentsurfuer your request is actually a duplicate of #312. At the moment we have other priorities that are ranked higher but this one is definitely not forgotten
